Makefile

```diff
@@ -145,6 +145,10 @@ fmt:
 	go fmt ./...
 	echo $(ALL_SRC) | xargs -n 10 $(GOIMPORTS) $(GOIMPORTS_OPT)
 
+.PHONY: fmt-sh
+fmt-sh:
+	shfmt -w -d -i 5 .
```

Comment:
Yes. That's correct. Added it to install tools and also added requirements for it. At first I should follow indents 2 to follow Google pattern. But 5 is better after looking at the results so that's why I used 5 instead of 2 (or 0 as default)